bundles/action/org.openhab.action.pushbullet/build.properties

```diff
@@ -0,0 +1,7 @@
+source.. = src/main/java/,\
+           src/main/resources/
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               about.html
```

Comment:
The OH2 static code checks require about.html to be distributed. Just in preparation for whenever OH1 will be statically checked as well 😄